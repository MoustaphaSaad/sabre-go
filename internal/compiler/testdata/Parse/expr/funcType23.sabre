func([]int, 5, "hello")