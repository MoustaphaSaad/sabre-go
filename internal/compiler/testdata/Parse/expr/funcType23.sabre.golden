>> 	func([]int, 5, "hello")
>> 	            ^           
Error[internal/compiler/testdata/Parse/expr/funcType23.sabre:1:13]: expected type but found LITERAL_INT(5)
>> 	func([]int, 5, "hello")
>> 	            ^           
Error[internal/compiler/testdata/Parse/expr/funcType23.sabre:1:13]: expected ')' but found 'LITERAL_INT'
